### PR TITLE
add ButtonAppBar

### LINKED ISSUE #1: As a user, I want to see the nav bar, so that the user can see the title
- [x] Install [MaterialUI](https://mui.com/)
- [x] Insert [App bar](https://mui.com/components/app-bar/) component

### PR DESCRIPTION
**Pull request by mpoloh**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+import ButtonAppBar from './ButtonAppBar';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
+      <header >
+        <ButtonAppBar />
+
```
Comment (i62navpm) [inline src/App.js]: Delete this blank space

File: src/ButtonAppBar.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            size="large"
+            edge="start"
+            color="inherit"
+            aria-label="menu"
+            sx={{ mr: 2 }}
+          >
+            <MenuIcon />
+          </IconButton>
+          <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+            News
```
Comment (i62navpm) [inline src/ButtonAppBar.js]: Replace "News" by "Events management" title

File: src/ButtonAppBar.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          </Typography>
+          <Button color="inherit">Login</Button>
```
Comment (i62navpm) [inline src/ButtonAppBar.js]: Remove this button

File: src/App.js (hunk 1 above)
Comment (mpoloh) [reply src/App.js]: There is no more white space

File: src/ButtonAppBar.js (hunk 2 above)
Comment (mpoloh) [reply src/ButtonAppBar.js]: Replace this title

File: src/ButtonAppBar.js (hunk 3 above)
Comment (mpoloh) [reply src/ButtonAppBar.js]: Ok, removed the button.

**Review by i62navpm — APPROVED** [review]

**Pull request merged**